templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -293,6 +296,9 @@ export class {{ service.name }}Client {
   {{ id.get("initialize") }}() {
     // If the client stub promise is already initialized, return immediately.
     if (this.{{ service.name.toCamelCase() }}Stub) {
+      {%- if service.options and service.options.deprecated %}
+      gax.warn('DEP1000', '{{service.name}} is deprecated and may be removed in a future version.', 'DeprecationWarning');
```

Comment:
Can we have a service name in the ID? Let's say there are two deprecated services, we want to receive separate warnings from all of them (but once for each).  We can make the ID look like `DEP${service.name}` for example.

Reply:
done